images/rabbitmq/Dockerfile

```
@@ -1,11 +1,25 @@
+FROM ${IMAGE_REPO:-lagoon}/commons as commons
 FROM rabbitmq:3-management
 
 ARG LAGOON_VERSION
 ENV LAGOON_VERSION=$LAGOON_VERSION
 
-COPY rabbitmq_delayed_message_exchange-3.7.0.ez /plugins
+ENV RABBITMQ_DEFAULT_USER='guest' \
+    RABBITMQ_DEFAULT_PASS='guest'\
+    RABBITMQ_DEFAULT_HA_PATTERN='^$'\
+    RABBITMQ_DEFAULT_VHOST='/'
+
+COPY --from=commons /bin/ep /bin/fix-permissions /bin/
 
+COPY rabbitmq_delayed_message_exchange-3.7.0.ez /plugins
 RUN rabbitmq-plugins enable --offline rabbitmq_delayed_message_exchange;
 
-ENV RABBITMQ_DEFAULT_USER=guest \
-    RABBITMQ_DEFAULT_PASS="guest"
+# Copy startup schema with vhost, users, permissions and policies
+COPY definitions.json /etc/rabbitmq/definitions.json
+RUN fix-permissions /etc/rabbitmq/definitions.json
+
+# Copy a custom entrypoint
+COPY cluster-rabbit.sh /
+RUN fix-permissions /cluster-rabbit.sh; chmod +x /cluster-rabbit.sh
```

```suggestion
RUN fix-permissions /cluster-rabbit.sh && chmod +x /cluster-rabbit.sh
```
let's always use `&&` instead of `;` for running multiple commands in a `RUN`, as the `;` will swallow any errors.